cmake/Modules/FindIrrlicht.cmake

```diff
@@ -0,0 +1,21 @@
+--I've got no clue how to get this working properly.
```

Comment:
Are there no existing cmake config files anywhere else on the internet? That's where I got some of the others! 😄 
If you can't find one I can fix this by copying and modifying another.

Reply:
I spent a few hours toying with the allegro configuration trying to get it to work for my renderer and finally gave up..